examples/e2e/consumerService.js

```diff
@@ -0,0 +1,8 @@
+let { server } = require('./consumer.js');
+
+server = server.listen(8080, () => {
```

Comment:
You can see that it's used below in outputting which host/ip it's bound to, but i'll simplify it.